CMakeLists.txt

```diff
@@ -545,7 +545,7 @@ message(STATUS "Using Lua interpreter: ${LUA_PRG}")
 
 option(COMPILE_LUA "Pre-compile Lua sources into bytecode (for sources that are included in the binary)" ON)
 
-if(COMPILE_LUA AND NOT WIN32)
+if(COMPILE_LUA AND CMAKE_BUILD_TYPE STREQUAL "Release" AND NOT WIN32)
```

Comment:
Should probably make this "not equal Debug" rather than "equal Release"

Reply:
Which side does RelWithDebug go?

Reply:
Hmmm. I think since it’s a release build it should still be enabled. It can always be disabled manually with the Cmake option

Reply:
Ok .

Reply:
Done.